pages/common/ssh-keygen.md

```diff
@@ -22,6 +22,10 @@
 
 `ssh-keygen -l -F {{remote_host}}`
 
+- Remove the keys of a host from your known_hosts file (useful when a known host has a new key):
```

Comment:
Sorry I missed it in the first pass, but we typically avoid mentioning the 2nd person in our pages. But I also see that the 4th example also has "your". Would be open to sending another change replacing "your" with the/one's ? It is completely fine if you don't want to. :)

Reply:
sure

Reply:
I've changed both instances